### PR TITLE
[OBSOLETE] Google Drive Persistence

### PR DESCRIPTION
**Pull request by jiachen247**
A client-side implementation of google drive apis to allow students to save and open files in the playground!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,10 @@
     }
   },
   "dependencies": {
+    "@types/gapi": "^0.0.36",
+    "@types/gapi.client": "^1.0.1",
+    "@types/gapi.client.drive": "^3.0.3",
+    "@types/google.picker": "^0.0.33",
```
Comment (geshuming) [inline package.json]: can we have @types dependencies saved as devDependencies? they're not needed for production

**Comment by martin-henz** [comment]
This project is shelved for 2020. Let's revisit this interesting feature and revive it then!

**Comment by martin-henz** [comment]
Let's try to reanimate this PR. 

Not surprising to see lots of conflicts.

Can we have a strategy how to proceed with this? Any suggestions anyone?

**Comment by martin-henz** [comment]
This PR is superseded by https://github.com/source-academy/cadet-frontend/pull/1083

**Pull request closed without merging**